tutorcruncher/hermes
# Company Custom Fields on Deals

Close #127 
## Technical Description
If a deal has a custom field with the same machine_name of a Company custom field, the deal custom field will inherit the value from the company custom field.

### Testing
- create custom field for deal with same machine_name to a company custom field, no hermes_name or tc2_machine_name is needed
- signup with tc, see Org and deal are created in PD with correct cf field values
- book a call, see org and deal are created in pd with correct cf field values
- update a 'inherited cf' on a deal in pipedrive, and it should do nothing to the org or other deals
- update the cf on the org and see it update all the inherited cfs on all the associated deals
